pkg/document/json/object.go

```diff
@@ -120,34 +124,47 @@ func (p *Object) SetNewTree(k string, initialRoot ...*TreeNode) *Tree {
 // SetNull sets the null for the given key.
 func (p *Object) SetNull(k string) *Object {
 	p.setInternal(k, func(ticket *time.Ticket) crdt.Element {
-		return crdt.NewPrimitive(nil, ticket)
```

Comment:
It will be nice to keep space convention of spacing before `return` in this file. I will leave only one comment for this, so check all the `return` codes in this file.